src/plugins/soundChanger/index.tsx

```diff
@@ -0,0 +1,130 @@
+let availableSounds: string[] = [];
+
+const settings = definePluginSettings({
+    // Detune sounds are alternative, weird sounding versions of the default sounds. I don't know why they exist
+    showDetuneSounds: {
+        type: OptionType.BOOLEAN,
+        description: "Show unused(?) \"detune\" sounds in SoundChanger settings. You probably don't need this enabled.",
+        default: false,
+        requiresRestart: true
+    },
+
+    replacements: {
+        type: OptionType.COMPONENT,
+        description: "",
+        component: () => {
+            const update = useForceUpdater();
+            return (
+                <>
+                    <Forms.FormTitle tag="h4">Sounds</Forms.FormTitle>
+                    <Flex flexDirection="column">
+                        {
+                            availableSounds.length > 0 ? availableSounds.map(sound => (
+                                <Flex
+                                    key={sound}
+                                    flexDirection="row"
+                                    style={{ alignItems: "center", justifyContent: "space-between" }}
+                                >
+                                    <Text>{sound.slice(2, -4)}</Text>
+                                    <TextInput
+                                        className="sound-changer-link-input"
+                                        value={soundReplacements.find(r => r.name === sound)?.link ?? ""}
+                                        placeholder="Link to a sound..."
+                                        onChange={link => {
+                                            const index = soundReplacements.findIndex(r => r.name === sound);
+
+                                            if (index === -1) {
+                                                soundReplacements.push({ name: sound, link });
+                                            } else {
+                                                soundReplacements[index].link = link;
+                                            }
+
+                                            DataStore.set("SoundChange_replacements", soundReplacements);
+                                            update();
+                                        }}
+                                    />
+                                </Flex>
+                            )) : <Text>Enable the plugin to see available sounds!</Text>
+                        }
+                    </Flex>
+                </>
+            );
+        }
+    }
+});
+
+export default definePlugin({
+    name: "SoundChanger",
+    authors: [Devs.SpikeHD],
+    description: "Change Discord sounds to be your own!",
+
+    patches: [{
+        find: "./message1.mp3",
+        replacement: [
+            // This only runs once, making it a good source for dynamically retrieving all of the sounds the user can change
+            {
+                match: /var (.{1,2})=(\{.+?\});/g,
```

Comment:
use \i to match identifiers instead of .{1,2}
```suggestion
                match: /var (\i)=(\{.+?\});/g,
```

Reply:
this match should also be made a bit more specific